src/main/java/core/basesyntax/Drawable.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface Drawable {
+    public abstract void draw();

```

Comment:
it's not necessary to add `public` and `abstract` to  interface methods, cause they are `public` and `abstract` by default
```suggestion
    void draw();
```